File: app/src/main/java/com/karthik/splash/Models/PhotoDetail/Location.java
package com.karthik.splash.Models.PhotoDetail;


import com.google.gson.annotations.SerializedName;


public class Location{

	@SerializedName("country")
	public String country;

	@SerializedName("city")
	public String city;

	@SerializedName("position")
	public Position position;
}

File: app/src/main/java/com/karthik/splash/models/photodetail/Location.kt
package com.karthik.splash.models.photodetail


import com.google.gson.annotations.SerializedName


data class Location(@SerializedName("country")
                    var country: String? = null,
                    @SerializedName("city")
                    var city: String? = null,
                    @SerializedName("position")
                    var position: Position? = null)